src/index.ts

```diff
@@ -149,6 +150,8 @@ export class FileSystem {
     )) as void;
   }
 
+  // remotePath: path of file to transfer in container
+  // localPath: path to save the transferred file on local machine
   static async transferFile(remotePath: string, localPath: string) {
```

Comment:
/**
 * Transfer file from container to host machine.
 * @param remotePath: file path in contianer workspace.
 * @param localPath: file path in host machine.
 */

Reply:
Fixed.